MarvelApiClient/Result.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Result.swift
+//  MarvelAPIClient
+//
+//  Created by Pedro Vicente Gomez on 05/01/16.
+//  Copyright © 2016 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamNetworking
+import Result
+import SwiftyJSON
+
+extension ResultType where Value == HTTPResponse {
```

Comment:
Ahhhhhh I was trying with Result instead of ResultType!!! Thanks Davide! Your Swift-Fu is over 9000!

![Over9000](http://orig08.deviantart.net/8580/f/2013/197/6/b/vegeta_shocked_9000_by_wortmann-d6drw47.gif)